populate/src/main/java/bio/terra/pearl/populate/dto/StudyEnvironmentPopDto.java

```diff
@@ -31,6 +31,8 @@ public class StudyEnvironmentPopDto extends StudyEnvironment {
     private List<String> kitTypeNames = new ArrayList<>();
     @Builder.Default
     private List<String> enrolleeFiles = new ArrayList<>();
+    @Builder.Default
+    private List<String> enrolleeRelationFiles = new ArrayList<>();
```

Comment:
Having these off the studyEnvPopDto will make these much easier to lose track of than if they were attached to the governedEnrollee.  I would put these on EnrolleePopDto instead.  Then you also won't need to have them be separate files, they can just be specified in the enrollee file. 

Populating will be a little trickier, since you need to make sure all enrollees are populated before populating the relations. so where you currently modified StudyPopulator to iterate over the enrolleeRelationFiles, instead you'll want to invoke a new method on enrolleePopulator

```
for (String enrolleeFile : studyPopEnv.getEnrolleeFiles()) {
            enrolleePopulator.populateRelations(context.newFrom(enrolleeFile), overwrite);
        }
```
and then that method in EnrolleePopulator can re-read the enrollee file, find any relations, and invoke EnrolleeRelationPopulator as needed